apps/journeys-admin/src/components/JourneyList/ActiveJourneyList/AddJourneyButton/AddJourneyButton.tsx

```diff
@@ -19,14 +22,18 @@ export function AddJourneyButton(): ReactElement {
   }
 
   return (
-    <Button
-      variant="contained"
-      startIcon={<AddIcon />}
-      size="medium"
-      onClick={handleClick}
-      sx={{ mt: 3, alignSelf: 'center' }}
-    >
-      Create a Journey
-    </Button>
+    <>
+      {activeTeam != null && (
```

Comment:
You need to consider the teams flag. So when the teams flag is true then we want to take into consideration the activeTeam otherwise we need to ignore it and show the button.

The hook needs to be added earlier:
```
const { teams } = useFlags()
```
```suggestion
      {(teams !== true || activeTeam != null) && (
```